xapian-applications/omega/index_file.cc

```diff
@@ -189,6 +190,63 @@ index_add_default_libraries()
     index_library("message/rfc822", omindex_gmime);
     index_library("message/news", omindex_gmime);
 #endif
+#if defined HAVE_LIBARCHIVE
+    Worker* omindex_libarchive = new Worker("omindex_libarchive");
+    index_library("application/vnd.oasis.opendocument.text",
+    omindex_libarchive);
```

Comment:
When wrapping a parameter list, Xapian-style is to line up the wrapped parameters under the first parameter (so the `o` should be directly below the opening `"`.

(I don't think I've seen the style you've used here before - it seems confusing as it makes it look like `omindex_libarchive);` is a separate statement.)

Reply:
Thanks for pointing this out. I will take care of this in future and have changed it.